CalcBinding.Core/CalcBinding.Core.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <TargetFrameworks>netcoreapp3.0;net45;net46</TargetFrameworks>
+    <UseWPF>true</UseWPF>
+    <ApplicationIcon />
+    <StartupObject />
+    <AssemblyName>CalcBinding</AssemblyName>
+    <RootNamespace>CalcBinding</RootNamespace>
+    <PackageId>CalcBinding.Core</PackageId>
+    <Version>2.4.0</Version>
+    <Authors>Alexander Zinchenko</Authors>
+    <Company>Alexander Zinchenko</Company>
+    <Product>CalcBinding.Core</Product>
+    <Description>.net core 3 port of the CalcBinding Library</Description>
+    <Copyright>Copyright © Alexander Zinchenko 2014-2018</Copyright>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageProjectUrl>https://github.com/bigworld12/CalcBinding</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/bigworld12/CalcBinding</RepositoryUrl>
+    <PackageTags>CalcBinding calc CalcBinding.Core core</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\CalcBinding\**\*.*">
+      <Link>%(RecursiveDir)%(FileName)%(Extension)</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="..\CalcBinding\obj\**" />
+    <Compile Remove="obj\**" />
+    <EmbeddedResource Remove="obj\**" />
+    <None Remove="obj\**" />
+    <Page Remove="obj\**" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="..\CalcBinding\CalcBinding.csproj" />
+    <Compile Remove="..\CalcBinding\CalcBinding.nuspec" />
+    <Compile Remove="..\CalcBinding\CreatePackage.bat" />
+    <Compile Remove="..\CalcBinding\Properties\AssemblyInfo.cs" />
+    <Compile Remove="..\CalcBinding\PublishPackage.bat" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DynamicExpresso.Core" Version="2.3.1" />
```

Comment:
CalcBinding also supports .net 4.0, version 2.3.1 of DynamicExpresso doesn't

Reply:
you can view problems if you switch Configuration to Debug-net4.0

Reply:
well, that's actually a problem, since only versions that support .net standard 2 are compiled against .net 4.5+, so i think that we might need to drop support for 4.0 to resolve this. but i think it's really ok since standard usage is leaning towards .net 4.5+ because win 8/10 is pre-installed with it, it's just that win 7 users need to update their framework version